go/ephemeral/storage.go

```diff
@@ -0,0 +1,218 @@
+package ephemeral
```

Comment:
This should be generic enough to reuse for user/deviceEK boxes. those storage levels will call out to the server for fill these caches and call the deviceEK store to unbox